src/Ting/Repository/Metadata.php

```diff
@@ -584,6 +584,28 @@ public function getByCriteria(
         return $queryGenerator->getByCriteria($criteriaColumn, $collectionFactory, $forceMaster);
     }
 
+    public function getByCriteriaWithOrderAndLimit(
+        array $criteria,
+        $orderBy,
```

Comment:
can you add the relevant type hints in the signature ?

Reply:
I added typehint for parameters `$order`,  `$limit` and `$forceMaster`